build/yaml/crd/nsx.vmware.com_vpcnetworkconfigurations.yaml

```diff
@@ -0,0 +1,140 @@
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: VPCNetworkConfigurationSpec defines the desired state of
+              VPCNetworkConfiguration. There is a default VPCNetworkConfiguration
+              that applies to Namespaces do not have a VPCNetworkConfiguration assigned.
+              When a field is not set in a Namespace's VPCNetworkConfiguration, the
+              Namespace will use the value in the default VPCNetworkConfiguration.
+            properties:
+              appliedToNamespaces:
+                description: Namespaces to be applied. The default VPCNetworkConfiguration
+                  will not have AppliedToNamespaces set.
+                items:
+                  type: string
+                type: array
+              defaultSubnetPrefixLength:
+                default: 28
+                description: Default subnet prefix length for Subnet. Defaults to
+                  28.
+                type: integer
+              defaultSubnetType:
+                default: public
+                description: DefaultSubnetType defines the type of the default SubnetSet
+                  for PodVM and VM. Must be public or private. Defaults to public.
+                enum:
+                - public
+                - private
+                type: string
+              edgeCluster:
+                description: Edge cluster path on which the networking elements will
+                  be created.
+                type: string
+              nsxtProject:
+                description: NSX-T Project the Namespace associated with.
+                type: string
+              privateCIDRs:
+                description: Private CIDRs used to allocate private Subnets.
+                items:
+                  type: string
+                maxItems: 5
+                minItems: 0
+                type: array
+              publicIPv4Block:
+                description: NSX-T IP Block paths used to allocate public Subnets.
+                items:
+                  type: string
+                maxItems: 5
+                minItems: 0
+                type: array
+              tier0Gateway:
+                description: Tier0 gateway or Tier0 VRF path.
+                type: string
+            type: object
+          status:
```

Comment:
Do we plan to set status for NetworkConfiguration CR?

Reply:
I plan to add validation for the vpcnetworkconfigurations, if there any validation errors, set the status to Ready:false.